bower.json

```
@@ -1,7 +1,7 @@
 {
   "name": "adapt-contrib-slider",
-  "version": "3.0.0",
-  "framework": ">=5",
+  "version": "3.0.1",
```

As this is more than just a 'bug fix'...
```suggestion
  "version": "3.1.0",
```